### DISCUSSION TITLE
Cardinal sequencer to Hardware Synths (solved)

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#252] [Q&A]
Hi,
I'm trying Cardinal with hardware synths today. It's very fun !!!
No problem with CV to notes C,C#, D... , no problem with CV to velocity or any Midi CC out to control the synths. 
To play notes on/off from Cardinal (seq module) to  hardware, some synths (ie:Novation Bass Station2, Mininova, Volca Drum, Volca Sample, TR8) recognize correctly the gates signals from a Cardinal sequencer module.
But others not (ie:Microfreaks, TD-3, TB-3, Volca Bass) ; with these synths, the Gate** signal is too short (like a TRIG signal) and I need to use ADDR-SEQ (It's a CV output) instead.

The ''problem'' is if I use ADDR-SEQ to play on-off notes, when I stop the DAW (and Cardinal), the last note stays sustained until I starts the Daw (and Cardinal).
Is it possible to stop/cut all CV when the DAW is stopped ? 

**(EDIT) I just fix the thing by adding VCV ADSR just after the gate sequencer. Now I just have to move the release to correct the time length of the gate signal to match the wish of the hardware synth. ;-)**

Thanks.
T.

**Comment by skeer562** [comment]
I'm wanting to get as far as you, but I'm not sure what steps to take.  What module did you use to to get midi to CV to control the hardware synths?  I have the Expert Sleepers ES-9 interface to get to modular.  How did you do the routing? Ex. connected patch cable from Cardinal Host Midi V/OCT Out to Cardinal Host Parameters Channel 1 input...(of course this doesn't work)...

**Reply by subderground** [reply]
Hello,
Personnaly, my routing is (cardinal vitual cv/gate - > midi cc/note) - > soundcard midi out - > hardware synth midi in. 
If I want to control a hardware cv with Cardinal, I need to plug the pc midi out to my Behringer Neutron witch have a "one paramter only'' midi to cv (mini jack out). 
I would love to bybass all that midi datas to have minijack cv outs directly on the pc to connect  to the hardware modular system... with a fair price 😉

Perhaps, my other post about controling Cardinal with another Cardinal instances on a different track would help you?! Reaper have audio in/out and cv in/out in its routing matrix. Perhaps, you could assign that kind of in/outs to es9?!
But, if you are on pc, I think your exert sleper es9 will be recognized as an audio interface. Sadly, pc users can't use several audio soundcards at the same time... except with the unstable workaround with asio4all. Btw if you find a solution on pc, please share; any cv/gate/trig bridge from Cardinal to eurorack would be great.
